parse_date.re

```diff
@@ -707,24 +707,25 @@ static const timelib_tz_lookup_table* abbr_search(const char *word, timelib_long
 	return NULL;
 }
 
-static timelib_long timelib_lookup_abbr(char **ptr, int *dst, char **tz_abbr, int *found)
+#define TIMELIB_TP_VALUE(tp) tp->gmtoffset - (tp->type * 3600)
```

Comment:
What does `TP_VALUE` mean? From the name I can't guess that it converts anything from hours to seconds for example.